src/framework/myCFG.ml

```diff
@@ -308,9 +384,22 @@ let createCFG (file: file) =
         let loop_heads = find_loop_heads_fun (module TmpCfg) fd in
         let reachable_return = find_backwards_reachable (module TmpCfg) (Function fd) in
```

Comment:
`reachable_from_return`?

Reply:
This was already here before, but the "backwards reachable from return" is just the optimal way to compute "nodes from which return is reachable" (one graph search instead of N). Naming it `reachable_from_return` would be misleading because everywhere else we think of reachability to mean forward reachability and nothing is forward reachable from the return node.